packages/spanner/src/transaction-request.js

```diff
@@ -203,11 +203,11 @@ TransactionRequest.prototype.createReadStream = function(table, query) {
 
   if (query.keys) {
     reqOpts.keySet = {
-      keys: [
-        {
-          values: arrify(query.keys).map(codec.encode)
-        }
-      ]
+      keys: arrify(query.keys).map(function(key) {
```

Comment:
This might be a mistake in the other direction? If I read this right, now _every_ key is treated as composite.
But maybe that is okay in Spanner? (Honestly asking.)